packages/client/src/apis/rewards/RewardsExecuteAPI.ts

```diff
@@ -93,33 +143,65 @@ export class RewardsExecuteAPI extends BaseVertexAPI {
   }
 
   /**
-   * Claim unlocked tokens that were previously unstaked
+   * Claim unlocked V2 tokens that were previously unstaked
    */
-  async withdrawUnstakedTokens() {
-    return this.context.contracts.vrtxStaking.write.claimVrtx();
+  async withdrawUnstakedV2Tokens() {
+    return this.context.contracts.vrtxStakingV2.write.claimWithdraw();
   }
 
   /**
-   * Claim unlocked V2 tokens that were previously unstaked
+   *
+   *  Staking Satellite
+   *
    */
-  async withdrawUnstakedV2Tokens() {
-    return this.context.contracts.vrtxStakingV2.write.claimWithdraw();
+
+  /**
+   * Stake V2 VRTX tokens on non-canonical chains
+   */
+  async stakeV2Satellite(params: CcipStakingParams) {
+    const address = this.getWalletClientAddress();
+
+    return this.context.contracts.vrtxStakingV2Satellite.write.stakeAs(
+      [address, toBigInt(params.amount)],
+      { value: toBigInt(params.ccipFee) },
+    );
   }
 
   /**
-   * Claim staking rewards (in USDC)
+   * Unstake V2 VRTX tokens on non-canonical chains, unstake tokens instantly with a penalty that is redistributed
+   * to the staking contract
    */
-  async claimStakingRewards() {
-    return this.context.contracts.vrtxStaking.write.claimUsdc();
+  async unstakeV2Satellite(params: CcipStakingParams) {
```

Comment:
Why does this take in `CcipStakingParams` when there is no need to input amount?